### ISSUE TITLE
Let users choose whether to open in new tab or new window by default

### ISSUE THREAD

**Issue by rolandcrosby** [rogual/neovim-dot-app#182]
In #117, @rogual wrote:
> Just checked MacVim again and it looks like it's a setting in preferences. (Called "open files from applications" but applies to file menu too). Happy to leave as is, maybe add a pref later.
I wanted to capture this in its own issue so it doesn't get lost. (As a side note, is there currently a good place for neovim-dot-app to put GUI-specific user preferences like this?)

**Comment by rogual** [comment]
Yes, this kind of thing is handled by functions defined in nvimrc. So we might have `MacSetOpenFilePreference('new-tab')` for example, and users would put that in their `vimrc`s.
